README.md

```diff
@@ -1 +1,19 @@
-# hungerrush_handheld_docs
\ No newline at end of file
+## Hugo documentation for Handheld
+
+This repository represents the application which is used by developers and key stakeholders to document the Hungerrush Hand Held platform.
+The documentation is written in Markdown and is rendered by [Hugo](https://gohugo.io/).
+The documentation is hosted on GitHub Pages and is available at [https://iiimpact.github.io/hungerrush_handheld_docs/](https://iiimpact.github.io/hungerrush_handheld_docs/).
+
+## Contributing
+
+Contributions are welcome. Please follow the setup and usage guides in this README to submit a pull request.
+a
+## Setup
+
+1. Install Hugo on your machine. See [Hugo installation guide](https://gohugo.io/getting-started/installing/).
+2. Clone this repository to your machine.
+3. Run `hugo server` in the root of the repository to start the Hugo server.
+4. Open [http://localhost:1313/dhungerrush_handheld_docs/](http://localhost:1313/hungerrush_handheld_docs/) in your browser to view the documentation.
```

Comment:
is the port number same on all machines (windows and mac)?

Reply:
yes, as far as I am aware it is :-)